packages/app/plugin/src/android/constants.ts

```diff
@@ -0,0 +1,7 @@
+export const DEFAULT_TARGET_PATH = 'app/google-services.json';
+
+// TODO: Is there a better place to store these values?
+// Maybe package.json?
+export const googleServicesClassPath = 'com.google.gms:google-services';
+export const googleServicesPlugin = 'com.google.gms.google-services';
+export const googleServicesVersion = '4.3.8';
```

Comment:
I saw some custom sections in `package.json` like `sdkVersions` - maybe there?

Reply:
https://github.com/invertase/react-native-firebase/blob/master/packages/app/package.json#L63 would probably be the best place for versions, versions there are also used in docs generation